### ISSUE TITLE
CUDA_ERROR_LAUNCH_FAILED

### ISSUE THREAD

**Issue by Robomate** [tensorflow/tensorflow#20356]
just wanna report this. it happened during the 60th call while I was calling a python script for about 100 times. (train 100 NNs).

system: gtx950m

at about the 30th call the neural network was not able to learn anymore. the accuracy and cost remained the same.

2018-06-27 21:10:37.421165: E tensorflow/stream_executor/cuda/cuda_driver.cc:1110] could not synchronize on CUDA context: CUDA_ERROR_LAUNCH_FAILED :: No stack trace available
2018-06-27 21:10:37.421173: E tensorflow/stream_executor/cuda/cuda_event.cc:49] Error polling for event status: failed to query event: CUDA_ERROR_LAUNCH_FAILED
2018-06-27 21:10:37.421198: F tensorflow/core/common_runtime/gpu/gpu_event_mgr.cc:203] Unexpected Event status: 1
Aborted (core dumped)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by Robomate** [comment]
I think this happens because I declare the model just once, even when I start the next training. 

Anyway just FYI, and I will close it now.

**Comment by JialieY** [comment]
I met the same issue. Anyone knows how to resolve it?
